user_manual/talk/advanced_features.rst

```diff
@@ -112,3 +112,44 @@ specific room.
 
 .. image:: images/talk-breakout-rooms-sidebar-item.png
 
+Call recording
+--------------
+Recording feature provides users an opportunity to:
+
+- Start and stop recordings during call.
+- Record video and audio stream of speaker, as well as screen share.
+- Access, share and download recorded files for future reference or distribution.
+
+Manage a recording
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The moderator of the conversation can start a recording together with a call start or anytime during it. To enable:
+
+- **Before the call**: tick a checkbox "Start recording immediately with the call" in Media settings, then click on "Start call".
+- **During the call**: click on the top-bar menu, then click "Start recording".
+
+.. image:: images/start-recording-before-call.png
+.. image:: images/start-recording-in-call.png
+
+The recording will start shortly, and you will see a red indicator next to the call time. By clicking on it, you can stop the recording any time.
+
+.. image:: images/stop-recording.png
+
+After stopping a recording, the server will take some time to prepare and save a recorded file.
+Moderator, who started the recording, should receive a notification about status of the file. From there it can be shared then to the conversation.
```

Comment:
```suggestion
The moderator, who started the recording, receives a notification when the recording is uploaded. From there it can be shared into the chat.
```